### ISSUE TITLE
Seaquest-v0 not training as well as announced

### ISSUE THREAD

**Issue by ThomasLecat** [dgriff777/rl_a3c_pytorch#11]
I have tried twice to train the agent on Seaquest-v0 with 32 workers on a server, but after 13 hours of training, the score seems to be stuck at 2700/2800 maximum.

Here's the log file : 
[log.txt](https://github.com/dgriff777/rl_a3c_pytorch/files/1244871/log.txt)

I'am using gym 0.8.1 and Atari-py 0.0.21 and let all the hyperparameters to their default value.
Any idea why the score obtained is much lower than the one you obtained ? (>50000)
Would you have the trained model for Seaquest-v0 ?
Thanks !

**Comment by ThomasLecat** [comment]
I have just watched the video created by gym-eval.py : what happens is that the agent kills all the enemies perfectly, but has never learned to surface !
Here's a gif:
![00000](https://user-images.githubusercontent.com/28880528/29610881-80083060-87f3-11e7-9c18-c1921dc4d5e1.gif)

Any idea why it stays at the bottom like that ?

**Comment by dgriff777** [comment]
Yup takes a long time for it to learn to surface correctly. Remember it took like nearly 24hrs then score started to shoot up. That game took nearly 3days if I remember correctly. Just let it keep training and it will sort itself out. As you can see it can shoot well so once the surfacing skill learned score should rise quickly after that

**Comment by ThomasLecat** [comment]
Got it, thanks ! Back to training then.
Do you think increasing the coefficient in front of the entropy term (currently 0.01) in the loss would make the agent discover the surface quicker by encouraging exploration ?

**Comment by dgriff777** [comment]
I think its good to leave the entropy term as is as its robust and effective for all games. I think massive exploration is one of the most important advantages of ai to humans so should not be neglected.

Added trained seaquest-v0 model to trained_models folder.

**Comment by ThomasLecat** [comment]
Ok for the entropy, and thanks for the trained model !
